README.md

```diff
@@ -287,6 +287,7 @@ Most configuration fields are named such that they can be defaulted to falsey. A
 | enableSessionStorageBuffer | true | Default true. If true, the buffer with all unsent telemetry is stored in session storage. The buffer is restored on page load |
 | isCookieUseDisabled | false | Default false. If true, the SDK will not store or read any data from cookies.|
 | cookieDomain | null | Custom cookie domain. This is helpful if you want to share Application Insights cookies across subdomains. |
+| cookiePath | null | Custom cookie path. This is helpful if you want to share Application Insights cookies behind an application gateway. |
 | isRetryDisabled | false | Default false. If false, retry on 206 (partial success), 408 (timeout), 429 (too many requests), 500 (internal server error), 503 (service unavailable), and 0 (offline, only if detected) |
```

Comment:
looks like you are still working on updating the docs. Are you planning to do this in a new docs pr?

Reply:
Yep, I've not completed the documentation yet

Reply:
Documentation is now ready for review